sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +148,68 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun cameraForAnnotations(
+    annotations: List<View>,
```

Comment:
With this approach we're forcing the consumers of this API to keep track of the view annotations themselves. They need to keep a [list](https://github.com/mapbox/mapbox-maps-android/pull/1753/files#diff-84626270966cd9b838ebc2ab46c6e4fb92f9354c0f9e9f821fa7813d18f2358aR30), [add](https://github.com/mapbox/mapbox-maps-android/pull/1753/files#diff-84626270966cd9b838ebc2ab46c6e4fb92f9354c0f9e9f821fa7813d18f2358aR91) and [remove](https://github.com/mapbox/mapbox-maps-android/pull/1753/files#diff-84626270966cd9b838ebc2ab46c6e4fb92f9354c0f9e9f821fa7813d18f2358aR96).

Is there any specific reason we can't use the list of views `ViewAnnotationsManagerImpl` keeps internally?

Reply:
Sorry I misunderstood it. Disregard this comment.